concepticondata/conceptlists/Mountain-1978-267.tsv

```diff
@@ -0,0 +1,268 @@
+Mountain-1978-267-153	153	ají	chili pepper	171	919	CHILI PEPPER
+Mountain-1978-267-154	154	coca	coca	172	137	COCA
+Mountain-1978-267-155	155	ayahuasca, yajé	Banisterium	173	276	BANISTERIUM
+Mountain-1978-267-156	156	plátano	plantain	174	399	PLANTAIN
+Mountain-1978-267-157	157	chonta	chonta palm	175	440	CHONTA PALM
+Mountain-1978-267-158	158	palma mirití	miriti palm	177		
+Mountain-1978-267-159	159	caña brava	cane	178	659	REED
+Mountain-1978-267-160	160	sal	salt	179	1274	SALT
+Mountain-1978-267-161	161	chicha, masato	chicha	180	441	CHICHA
+Mountain-1978-267-162	162	uno	one	181	1493	ONE
+Mountain-1978-267-163	163	dos	two	182	1498	TWO
+Mountain-1978-267-164	164	sonajero, maraca	rattle	193	300	RATTLE
+Mountain-1978-267-165	165	tambor	drum	194	908	DRUM
+Mountain-1978-267-166	166	orejeras, aretes	ear ornament	196		
+Mountain-1978-267-167	167	yo	I	200	1209	I
+Mountain-1978-267-168	168	tú	thou	201	1215	THOU
+Mountain-1978-267-169	169	nosotros	we	205	1212	WE
+Mountain-1978-267-170	170	grande	big	220	1202	BIG
+Mountain-1978-267-171	171	frío	cold	222	1287	COLD
+Mountain-1978-267-172	172	caliente	hot	223	1286	HOT
+Mountain-1978-267-173	173	bueno	good	224	1035	GOOD
+Mountain-1978-267-174	174	blanco	white	226	1335	WHITE
+Mountain-1978-267-175	175	negro	black	227	163	BLACK
+Mountain-1978-267-176	176	¡vaya!, ¡ve!	Go!	228		
+Mountain-1978-267-177	177	¡ven!	Come!	229		
+Mountain-1978-267-178	178	¡come!	Eat!	230		
+Mountain-1978-267-179	179	¡bebe!	Drink!	231		
+Mountain-1978-267-180	180	¡duerme!	Sleep!	232		
+Mountain-1978-267-181	181	punta de la lengua	tip of tongue	235	443	TIP (OF TONGUE)
+Mountain-1978-267-182	182	pelo largo	long hair	236	444	LONG HAIR
+Mountain-1978-267-183	183	toda la garganta y cuello	around the neck	237		
+Mountain-1978-267-184	184	antebrazo	lower arm	240	446	LOWER ARM
+Mountain-1978-267-185	185	la pierna inferior	lower leg	242	447	LOWER LEG
+Mountain-1978-267-186	186	intestinas	intestines	245	1991	INTESTINES
+Mountain-1978-267-187	187	nubes quietas	clouds at rest	247	1489	CLOUD
+Mountain-1978-267-188	188	nubes de tormenta	storm clouds	248	448	STORM CLOUDS
+Mountain-1978-267-189	189	chagra	field	249	212	FIELD
+Mountain-1978-267-190	190	quebrada	stream	250	947	STREAM
+Mountain-1978-267-191	191	piedrecillas	pebbles	251	961	PEBBLES
+Mountain-1978-267-192	192	rocas	huge rocks	252		
+Mountain-1978-267-193	193	tambo	shelter	255		
+Mountain-1978-267-194	194	este	this	256	1214	THIS
+Mountain-1978-267-195	195	aquel	that	257	78	THAT
+Mountain-1978-267-196	196	quien	who	258	1235	WHO
+Mountain-1978-267-197	197	que	what	259	1236	WHAT
+Mountain-1978-267-198	198	no	not	260	1240	NOT
+Mountain-1978-267-199	199	todos	all	261	98	ALL
+Mountain-1978-267-200	200	muchos	many	262	1198	MANY
+Mountain-1978-267-201	201	largo	long	263	1203	LONG
+Mountain-1978-267-202	202	corteza	bark	264	1204	BARK
+Mountain-1978-267-203	203	carne	flesh	265	2615	FLESH OR MEAT
+Mountain-1978-267-204	204	grasa	grease	267	323	FAT (ORGANIC SUBSTANCE)
```

Comment:
FAT (FROM ANIMALS) in Huber.

Reply:
Yes, but there the English gloss is different to the SIL list I used here. Does grasa actually imply being from animals?

Reply:
I think, grasa can be used more broadly now, but it is still predominantly used to denote to fat from animals (or humans), so in some sense as some kind of body part.